### PR TITLE
Fixes #8672: Add a technique specific scenario to rtf

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/8672

### CODE REVIEW

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
Commit modified

File: scenario/lib.py [hunk 1]
```diff
@@ -175,3 +177,35 @@ def host_date(name, error_mode, server):
...
+      match = re.search(r'^([\w\-]+)\.json$', file)
+      if not match:
+        continue
+      directive = root + "/" + file
+      metadata = root + "/" + match.group(1) + ".metadata"
+      if not os.path.isfile(metadata):
+        metadata = None
+      test = root + "/" + match.group(1) + ".rb"
+      if not os.path.isfile(test):
+        continue
```
Comment (amousset) [inline scenario/lib.py]: Shouldn't missing json or rb file be an error? Skipping may hide a broken test.

**Comment by amousset** [comment]
Looks good! (apart from the incomplete platform file and missing doc).

File: scenario/lib.py (hunk 1 above)
Comment (peckpeck) [inline scenario/lib.py]: indeed

**Comment by peckpeck** [comment]
Commit modified

File: platforms/all_agents.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "default":{ "run-with": "vagrant", "rudder-version": "3.2", "system": "ubuntu14_04", "inventory-os": "ubuntu" },
+  "server": { "rudder-setup": "server" },
+  "agent1": { "rudder-setup": "agent", "system": "ubuntu14_04", "inventory-os": "ubuntu" },
```
Comment (amousset) [inline platforms/all_agents.json]: This one appears twice.

File: platforms/all_agents.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  "agent1": { "rudder-setup": "agent", "system": "centos6", "inventory-os": "centos" },
+  "agent1": { "rudder-setup": "agent", "system": "centos7", "inventory-os": "centos" },
+  "agent1": { "rudder-setup": "agent", "system": "debian6", "inventory-os": "debian" },
+  "agent1": { "rudder-setup": "agent", "system": "debian7", "inventory-os": "debian" },
+  "agent1": { "rudder-setup": "agent", "system": "debian8", "inventory-os": "debian" },
+  "agent1": { "rudder-setup": "agent", "system": "sles11", "inventory-os": "suse" },
+  "agent1": { "rudder-setup": "agent", "system": "sles12", "inventory-os": "suse" },
+  "agent1": { "rudder-setup": "agent", "system": "ubuntu10_04", "inventory-os": "ubuntu" },
+  "agent1": { "rudder-setup": "agent", "system": "ubuntu12_04", "inventory-os": "ubuntu" },
+  "agent1": { "rudder-setup": "agent", "system": "ubuntu14_04", "inventory-os": "ubuntu" }
```
Comment (amousset) [inline platforms/all_agents.json]: We probably want different names for the VMs.

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**